docs/card.md

```diff
@@ -3,10 +3,6 @@ id: card
 title: Card
 ---
 
-Cards are a great way to display information, usually containing content and
```

Comment:
Why'd you remove this?

Reply:
damn it seems like I compared with master sorry it is fixed